core/src/components/checkbox/test/label/checkbox.e2e.ts

```diff
@@ -2,20 +2,18 @@ import { expect } from '@playwright/test';
 import { configs, test } from '@utils/test/playwright';
 
 /**
- * By default ion-checkbox only takes up
- * as much space as it needs. Justification is
- * used for when the checkbox takes up the full
- * line (such as in an ion-item). As a result,
- * we set the width of the checkbox so we can
- * see the justification results.
+ * By default ion-checkbox only takes up as much space
+ * as it needs. Justification is used for when the
+ * checkbox should take up the full line (such as in an
+ * ion-item or when it has 100% width).
  */
 configs().forEach(({ title, screenshot, config }) => {
   test.describe(title('checkbox: label'), () => {
     test.describe('checkbox: start placement', () => {
       test('should render a start justification with label in the start position', async ({ page }) => {
         await page.setContent(
           `
-           <ion-checkbox label-placement="start" justify="start" style="width: 200px">Label</ion-checkbox>
```

Comment:
An explicit width is no longer needed since using `justify` sets `display: flex` which takes up the full-width of its container.